modules/ui/panels/history.js

```diff
@@ -49,7 +49,7 @@ export function uiPanelHistory(context) {
             .attr('href', 'https://hdyc.neis-one.org/?' + userName)
             .attr('target', '_blank')
             .attr('tabindex', -1)
-            .html('HDYC');
+            .html('Profile');
```

Comment:
As I belatedly realized in https://github.com/openstreetmap/iD/issues/8784#issuecomment-958186532, on each row, each link displays similar information using different tools. So there’s already a different profile link directly to the left of this link. To avoid confusion, we can call `links.html(t('info_panels.history.profile'))` a few lines above, before appending the first link, then leave this link as it was.